### ISSUE TITLE
Inconsistent title/logo

### ISSUE THREAD

**Issue by eyoder** [vjeantet/hugo-theme-docdock#125]
Love the improvements to the site, but I found an inconsistency:

In /single.html and /baseof.html pages, the logo section displays the site name. But in /li.html pages, .Title is displayed, and it's a link, and a relatively large font size < h2 >, vs < p >.

![image](https://user-images.githubusercontent.com/5911759/35530266-733941b8-04e8-11e8-84af-03882ad08d1d.png)

**Comment by awgeorge** [comment]
Seeing this too - have you worked out why, so we can submit a PR.

**Comment by eyoder** [comment]
Yeah, no, not a clue. I'm a tech writer, so I know enough css to be dangerous.

**Comment by codenesium** [comment]
I'm seeing this as well. I'll try to figure out what's causing it. Great template other than this little bug.

**Comment by codenesium** [comment]
I was able to reproduce this by having a blank _index.md page in the directory structure. Adding anything below the +++ makes the problem go away, 

```
+++
title = "Codenesium Documentation"
description = ""
+++
```

**Comment by bsc7080hum** [comment]
Is the only fix presently just swapping out to </div> ?
